django/core/cache/backends/memcached.py

```diff
@@ -174,6 +185,15 @@ def _cache(self):
         return self._lib.Client(self._servers, **self._options)
 
     def close(self, **kwargs):
-        # libmemcached manages its own connections. Don't call disconnect_all()
+        # libmemcached managepip install pipenvs its own connections. Don't call disconnect_all()
```

Comment:
"managepip"?

Reply:
oh nuts, I accidentally pasted that there and didn't spot it, I'll remove it.